src/components/SidePanel.js

```diff
@@ -118,6 +119,26 @@ function SidePanel(props) {
     }
   }
 
+  function calculateLatLonDistance(capture1Id, capture2Id) {
+    let obj1 = verifyContext.captureImages.filter(
+      (capture) => capture.id == capture1Id
+    )[0];
+    let obj2 = verifyContext.captureImages.filter(
+      (capture) => capture.id == capture2Id
+    )[0];
+    const distance = getDistance(
+      {
+        latitude: Number(obj1.lat),
+        longitude: Number(obj1.lon),
+      },
+      {
+        latitude: Number(obj2.lat),
+        longitude: Number(obj2.lon),
+      }
+    );
+    return `Distance bewteen selected captures: ${distance.toLocaleString()}m`;
```

Comment:
Let's limit this distance to one decimal place.

Reply:
Done!